lochness/redcap/__init__.py

```diff
@@ -333,12 +333,25 @@ def sync(Lochness, subject, dry=False):
             logger.debug("Downloading REDCap data")
             _debug_tup = (redcap_instance, redcap_project, redcap_subject)
 
-            record_query = {
-                'token': api_key,
-                'content': 'record',
-                'format': 'json',
-                'records': redcap_subject
-            }
+            if 'UPENN' in redcap_instance:
+                # UPENN REDCap is set up with its own record_id, but have added
+                # "session_subid" field to note AMP-SCZ ID
+                redcap_subject_sl = redcap_subject.lower()
+                record_query = {
+                    'token': api_key,
+                    'content': 'record',
+                    'format': 'json',
+                    'filterLogic': f"[session_subid] = '{redcap_subject}' or "
+                                   f"[session_subid] = '{redcap_subject_sl}'"
```

Comment:
Does this assignment work in Python? :o

Reply:
Yes, it works. When the line breaks are within `{}`, `()`, or `[]`, you do not have to add `\` at the end of the string to concatenate them.

Reply:
Right I meant does this complex query work for REDCap?

Reply:
Yes. It does.